addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -420,10 +420,11 @@ def migration_invoice_moves(env):
     )
     openupgrade.logged_query(
         env.cr, """
-        UPDATE account_invoice_payment_rel aipr
-        SET invoice_id = am.id
-        FROM account_move am
-        WHERE am.old_invoice_id = aipr.invoice_id
+        INSERT INTO account_invoice_payment_rel
```

Comment:
If you do this (which is not incorrect at all), you should rename the old table in pre-migration for not twisting references in the same table.

Reply:
I already did that https://github.com/OCA/OpenUpgrade/pull/2919/files#diff-056cf6d26f1211252b7063df64efb06222e4797b0973bf784c70b5d3e25d25ceR54

Reply:
Oh, I didn't see it, sorry.

Reply:
Don't worry, too many lines to review, so that could happen to anyone :coy: